be/src/storage/tablet_updates.cpp

```diff
@@ -2849,13 +2881,16 @@ void TabletUpdates::get_tablet_info_extra(TTabletInfo* info) {
         LOG_EVERY_N(WARNING, 10) << "get_tablet_info_extra() some rowset stats not found tablet=" << _tablet.tablet_id()
                                  << " rowset=" << err_rowsets;
     }
+    int64_t pindex_size = 0;
+    int64_t col_size = 0;
+    (void)_get_extra_file_size(&pindex_size, &col_size);
     info->__set_version(version);
     info->__set_min_readable_version(min_readable_version);
     info->__set_max_readable_version(max_readable_version);
     info->__set_version_miss(has_pending);
     info->__set_version_count(version_count);
     info->__set_row_count(total_row);
-    info->__set_data_size(total_size);
+    info->__set_data_size(total_size + pindex_size + col_size);
```

Comment:
How about separating the data size from the size of other files? Maybe users want to discern the specific size of the data.

Reply:
User use `show data` to display the space cost. i can add more fields for system table, but I think data size should contains all.